### ISSUE TITLE
Unexpected `spaceInsideBraces` spacing instead of newline

### ISSUE THREAD

**Issue by keith** [nicklockwood/SwiftFormat#652]
With this code:

```swift
foo.subscribe(onNext: {
       if true {
       }})
```

If you format with `swiftformat --rules spaceInsideBraces foo.swift --swiftversion 5.2 --cache clear`

You get:

```swift
foo.subscribe(onNext: {
       if true {
       } })
```

But maybe this should be used alongside another rule to give you something more like:

```swift
foo.subscribe(onNext: {
       if true {
       }
})
```

0.44.13

**Comment by nicklockwood** [comment]
> But maybe this should be used alongside another rule to give you something more like:
>
> ```
> foo.subscribe(onNext: {
>        if true {
>        }
> })
> ```

With the default settings, the `emptyBraces` rule means this gets formatted as

```swift
foo.subscribe(onNext: {
    if true {}
})
```

which seems pretty reasonable.

**Comment by nicklockwood** [comment]
@keith fixed in 0.44.15. Thanks for reporting!
